hera_cal/lstbin_simple.py

```diff
@@ -2037,4 +2119,5 @@ def lst_bin_arg_parser():
     a.add_argument("--no-flagged-mode", action='store_true', help="turn off output of flagged mode LST-binning")
     a.add_argument("--do-inpaint-mode", action='store_true', default=None, help="turn on inpainting mode LST-binning")
     a.add_argument("--where-inpainted-file-rules", nargs='*', type=str, help="rules to convert datafile names to where-inpainted-file names. A series of two strings where the first will be replaced by the latter")
+    a.add_argument('--sigma-clip-in-inpainted-mode', action='store_true', default=False, help='allow sigma-clipping in inpainted mode')
```

Comment:
Were you planning on also adding mad/med options here?

Reply:
Yes!